### ISSUE TITLE
RAM crashes when generating mask using xml file

### ISSUE THREAD

**Issue by LiangJunhao-THU** [dong100136/Breast-Cancer-Image-Classification-On-WSI-With-Spatial-Correlations#2]
A fantastic job!
But when I running your code to generate the region mask, the service raised my RAM is not sufficient. By the way, I run this code on the Colab.
![7YH6{0W0KR}36WE1J$O)F1W](https://user-images.githubusercontent.com/41545346/79113204-f669c900-7db2-11ea-8343-31d38570d522.png)
![(71)1B7@ BS7R 9S8YUS7YK](https://user-images.githubusercontent.com/41545346/79113211-f9fd5000-7db2-11ea-874e-9a8052053fef.png)
Your work says "we implemented with Keras-2.0.8 and Tensorflow-1.4.1 in python 3.6 with NVIDIA GeForce GTX 1080 Ti GPU", but is there any request for RAM for running the code?
I guess it is very hard for Colab to generate WSIs masks, which is so big. May you tell me some solutions to deal with this question?
Thanks!
